$NetBSD: patch-cn,v 1.3 2012/11/13 21:23:07 adam Exp $

The mem-overflow test contains a declaration of an empty struct, which breaks
on at least SunPro cc, and presumably some other non-gcc compilers.  Skip this
test for now, pending a better fix.
https://bugzilla.gnome.org/show_bug.cgi?id=641350

--- glib/tests/Makefile.am.orig	2011-06-05 23:18:49.000000000 +0000
+++ glib/tests/Makefile.am
@@ -77,9 +77,6 @@ hostutils_LDADD     = $(progs_ldadd)
 TEST_PROGS         += gvariant
 gvariant_LDADD      = $(progs_ldadd)
 
-TEST_PROGS         += mem-overflow
-mem_overflow_LDADD  = $(progs_ldadd)
-
 TEST_PROGS              += utf8-performance
 utf8_performance_SOURCES = utf8-performance.c
 utf8_performance_LDADD   = $(progs_ldadd)
